.github/workflows/workflow.yml

```
@@ -128,8 +128,8 @@ jobs:
     needs: [ check-for-functional-changes ]
     if: needs.check-for-functional-changes.outputs.status == 'detected_functional_changes'
     env:
-      PYPI_USERNAME: openfisca-bot
-      PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      PYPI_USERNAME: __token__
+      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
```

As this token is linked to a personal PyPi account, I suggest to use the token of a shared PyPi account (otherwise, we might have some blocking situations when our colleague isn't available 😊).
 ```suggestion
      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN_OPENFISCA_BOT }}
```